baseapp-auth/baseapp_devices/graphql/queries.py

```diff
@@ -0,0 +1,11 @@
+import graphene
+
+from baseapp_devices.graphql.object_types import UserDeviceType
+from baseapp_devices.models import UserDevice
+
+
+class UserDeviceQueries:
+    all_user_devices = graphene.List(UserDeviceType)
```

Comment:
lets use `DjangoConnectionField` instead to make this paginated.

and maybe (just a thought/suggestion) we can add this to the `UserObjectType` instead of having it as a query, this way you could query for it like this:
```
{
   me {
      devices {
         ....
      }
  }
}
```
i think it follows more the idea of a "graph"